content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_dropship.rst

```diff
@@ -0,0 +1,263 @@
+add a new component, click :guilabel:`Add a line`, select the component in the :guilabel:`Component`
+drop-down menu, and specify the required quantity in the :guilabel:`Quantity` field.
+
+Configure Components
+--------------------
+
+To configure components for the *Dropship Subcontractor on Order* route, navigate to each component
+from the |BoM| by selecting the component's name in the :guilabel:`Components` tab, and clicking the
+:guilabel:`➡️ (right arrow)` button to the right of the name.
+
+Alternatively, navigate to each component by going to :menuselection:`Inventory app --> Products -->
+Products`, and selecting the component.
+
+On the component product form, select the :guilabel:`Purchase` tab, and add a vendor by clicking
+:guilabel:`Add a line`, selecting the vendor in the :guilabel:`Vendor` field, and adding the price
+they sell the product for in the :guilabel:`Price` field. This is the vendor that sends components
+to the subcontractor, once they are purchased.
+
+Then, click on the :guilabel:`Inventory` tab and select the :guilabel:`Dropship Subcontractor on
+Order` route in the :guilabel:`Routes` section.
+
+Repeat the process for every component that must be dropshipped to the subcontractor.
+
+.. _manufacturing/workflows/subcontracting-dropship:
+
+Dropship subcontractor on order workflow
+========================================
+
+The dropship subcontractor on order workflow consists of up to six steps:
+
+#. Create a sales order (SO) for the subcontracted product; doing so creates a *subcontractor* |PO|
+   to purchase the product from the subcontractor.
+#. Confirm the |PO| created in the previous step, or create a new |PO|; doing so creates a request
+   for quotation (RfQ) to purchase the components from the vendor, as well as a receipt order or a
+   dropship order.
+#. Confirm the |RfQ| to turn it into a second |PO| (*vendor* |PO|); doing so creates a *Dropship
+   Subcontractor* order.
+#. Process the *Dropship Subcontractor* order once the vendor has sent the components to the
+   subcontractor.
+#. Process the receipt once the subcontractor has finished manufacturing the subcontracted product,
+   and shipped it back to the contracting company **OR** process the dropship order to ship the
+   product directly to the end customer.
+#. If the workflow was started by creating an |SO|, and the finished product is not dropshipped to
+   the end customer, process the delivery order once the product has been shipped to the customer.
+
+The specific number of steps depends on the reason that the subcontracted product is being purchased
+from the subcontractor.
+
+If the reason is to fulfill a specific customer order, the process starts with creating an SO, and
+ends with delivering the product to the customer, or having the subcontractor dropship it to them.
+
+If the reason is to increase quantity of stock on-hand, the process starts with creating a PO, and
+ends with receiving the product into inventory.
```

Comment:
Great concluding statements to this section, covering the range of use cases here 👍